docs/JAX_porting_HF_model.md

```diff
@@ -0,0 +1,453 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
++++ {"id": "wXMWNV2Rw-wE"}
+
+# Converting the LLama 3 8B Instruct model from Hugging Face to JAX
+
+[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/jax-ml/jax-ai-stack/blob/main/docs/convert_from_HF.ipynb)
+
+[![Kaggle](https://kaggle.com/static/images/open-in-kaggle.svg)](https://github.com/jax-ml/jax-ai-stack/blob/main/docs/JAX_for_LLM_pretraining.ipynb)
+
+This tutorial demonstrates how to you could convert Meta's [Llama 3 8B Instruct model](https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct) from Hugging Face to a JAX model and run it on CPU.
+
+Since the model is fairly large, you are going to need either **a Colab's high RAM VM** (which requires Pro subscription) or **a Kaggle VM**.
```

Comment:
I think it'll be helpful to include docs links for both of these setups.

Reply:
There isn't much to link to actually, only the Colab pricing page.